manifest/usr/bin/dmarcts-report-parser.conf

```diff
@@ -12,8 +12,8 @@ $dbname = $ENV{'REPORT_DB_NAME'};
 $dbuser = $ENV{'REPORT_DB_USER'};
 $dbpass = $ENV{'REPORT_DB_PASS'};
 $dbhost = $ENV{'REPORT_DB_HOST'}; # Set the hostname if we can't connect to the local socket.
-$dbport = $ENV{'REPORT_DB_PORT'} // 3306;
```

Comment:
That's not good, the current behaviour is at least setting this port always to the mysql one 3306 - every user would on the next image update have a [parser logic](https://github.com/techsneeze/dmarcts-report-parser/blob/master/dmarcts-report-parser.pl#L273) change that the DBI connect call uses `port=` instead of `port=3306`. Might not be fatal, but it would be a bad change to introduce.